id-authentication-default.properties

```diff
@@ -403,9 +403,9 @@ ida.api.version.kycexchange=1.0
 static.token.enable=true
 
 ## Allowed ID Types (allowed values : UIN/VID/USERID) to be supported for Authentication/KYC/OTP Requests
-request.idtypes.allowed=VID,UIN
```

Comment:
Why UIN removed? This property will be application to normal authentication as well.